apps/workers/db-check/worker.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math"
+	"time"
+
+	"github.com/jbsmith7741/uri"
+	"github.com/pcelvng/task"
+	"github.com/pcelvng/task-tools/slack"
+)
+
+type worker struct {
+	options
+	task.Meta
+	slack *slack.Slack
+
+	Table     string    `uri:"table" required:"true"`      // name of the table to query
+	DateField string    `uri:"date_field" required:"true"` // the date field to get row counts
+	Date1     time.Time `uri:"date1" format:"2006-01-02"`  // default is 24h back from today
+	Date2     time.Time `uri:"date2" format:"2006-01-02"`  // 2nd date to compare records against date1
+	Offset    string    `uri:"offset"`                     // duration back from date1 used to determine the value of date2 if needed
```

Comment:
change the offset type to a `time.Duration`

Reply:
with uri translate a string into a duration? I think you mentioned that before. That would be nice.